launchable/commands/record/build.py

```diff
@@ -283,7 +283,7 @@ def _get_branch_name(repo_dist: str) -> str:
     try:
         refs = subprocess.check_output(
             "git show-ref | grep '^'$(git rev-parse HEAD)",
-            cwd=repo_dist).decode().split("\n")
+            cwd=repo_dist, shell=True).decode().split("\n")
```

Comment:
[must]
`shell=True` option is known to be unsecure for shell injection vulnerability.
Please find and try other way.

https://docs.python.org/ja/3/library/subprocess.html#security-considerations
https://www.codiga.io/blog/python-subprocess-security/
https://medium.com/@acharya.vikash/avoid-using-shell-true-in-python-subprocess-module-e95fed487f19

Reply:
Thank you, I'll fix it!